### ISSUE TITLE
[Python] exception handling issue

### ISSUE THREAD

**Issue by nim65s** [stack-of-tasks/pinocchio#1075]
Hi,

https://github.com/stack-of-tasks/pinocchio/blob/7765909f1aed41dd9471baaf8e492be5fcc99523/examples/display-shapes.py#L40

On my system, this leads to the following error:
```
Error while initializing the viewer. It seems you should install gepetto-viewer
Traceback (most recent call last):
  File "…/pinocchio-2.3.0/examples/display-shapes.py", line 37, in <module>
    viz.initViewer()
  File "…/pinocchio-2.3.0/bindings/python/pinocchio/visualize/gepetto_visualizer.py", line 28, in initViewer
    import gepetto.corbaserver
ModuleNotFoundError: No module named 'gepetto'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "…/pinocchio-2.3.0/examples/display-shapes.py", line 40, in <module>
    print(error.message)
AttributeError: 'ModuleNotFoundError' object has no attribute 'message'
```

I can probably fix this if no one else is available.

**Comment by jcarpent** [comment]
Please go ahead with the fix. Thanks in advance @nim65s.

**Comment by jcarpent** [comment]
@nim65s I plan to make a new release with bug fixes. Could you handle this issue today?

**Comment by jcarpent** [comment]
In fact, I will handle it by myself

**Comment by jcarpent** [comment]
Solved by #1081.
